site/docs/components/range-slider/examples.mdx

```diff
@@ -20,13 +20,13 @@ Selection of an interval of values from a `RangeSlider` track. A tooltip display
 
 ### Best practices
 
-When adding custom text for labelling minimum and maximum values on the slider, either via `minLabel` and `maxLabel` props or via `marks`, we recommend adding corresponding `accessibleMinText` and `accessibleMaxText` so that the screen reader can make accurate announcements for enhanced accessibilty.
+To make the `RangeSlider` accessible, ensure screen readers can accurately convey the range and its values. While screen readers announce the default minimum and maximum values, customizing labels with `minLabel` and `maxLabel` is best paired with updating screen reader text using `accessibleMinText` and `accessibleMaxText`. These props let you define the exact text announced for the minimum and maximum values, ensuring alignment with custom labels. For instance, if `minLabel` is "Low" and `maxLabel` is "High," set `accessibleMinText` to "Low" and `accessibleMaxText` to "High" for a more intuitive experience for assistive technology users.
```

Comment:
in the rest of your changes, you made aria labels lowercase, but we kept this as Titlecase ?